io.github.magnush.mhWaveEdit.yml

```diff
@@ -0,0 +1,107 @@
+# io.github.magnush.mhWaveEdit.yml
+app-id: io.github.magnush.mhWaveEdit
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: mhwaveedit
+
+#separate-locales: false
+
+rename-desktop-file: mhwaveedit.desktop
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-music
+  - --socket=pulseaudio
+  
+modules:
+
+
+- name: gtk2
```

Comment:
you should use the shared-modules for this

Reply:
using shared modules